usaspending_api/submissions/helpers.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date
+
+from usaspending_api.submissions.models import DABSSubmissionWindowSchedule
+
+
+def get_last_closed_submission_date(is_quarter: bool) -> dict:
+    current_date = date.today()
```

Comment:
What prompted the change from datetime.now() to date.today()?

Since these are local times, it is always a good idea to make them TZ-aware like `datetime.now(timezone.utc)`

Reply:
Didn't think about that but can make the change.